_services_projects/legislation-as-code.md

```diff
@@ -6,7 +6,7 @@ permalink: /projects/legislation-as-code/
 redirect_from: /projects/legislation-as-code/
 excerpt: Exploring the benefits of legislation that is both human and machine readable and what it might take to transition.
 image: /assets/img/projects/legislation-as-code.png
-image_accessibility: 
+image_accessibility: Image of OpenFisca & how it calculates an output using rules & formulas.
```

Comment:
Can we describe how Open Fisca calculates an output using rules and formulas? To ensure the blind user gets the same information a sighted user would.